releasenotes/notes/4872-Deprecate-Schedule-input-arg-schedules.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I couldn't understand it, where do you take the SHA appended to the releasenotes' filenames?

Reply:
DId you use the reno CLI tool?
```
reno new 4872-deprecate-...
```

Reply:
Ah, I had missed that! Thanks :-)

Reply:
Ah, I had missed that! Thanks :-)